nbs/utils.ipynb

```diff
@@ -366,12 +366,173 @@
     "ax.grid()"
```

Comment:
Is it possible to use show_doc on the new functions for the calendar features documentation?